master/internal/api_workspace.go

```diff
@@ -527,7 +527,7 @@ func (a *apiServer) deleteWorkspace(
 	log.Debugf("deleting workspace %d projects", workspaceID)
 	holder := &workspacev1.Workspace{}
 	for _, pj := range projects {
-		expList, err := a.m.db.ProjectExperiments(int(pj.Id))
+		expList, err := db.ProjectExperiments(context.TODO(), int(pj.Id))
```

Comment:
using `context.TODO()` here because otherwise I was running into context cancelled errors; I _think_ that `defer rows.Close()` within `ProjectExperiments(...)` might be cancelling the context